README.md

```diff
@@ -1,10 +1,11 @@
 # BotKube
```

Comment:
After a bit of thinking I would vote for replacing `infracloudio` to `kubeshop` in CHANGELOG.md 🙂 For older BotKube versions the old links, license etc. will still stay there on a specific tags, so let's straighten it out for all future releases.